packages/core/src/components/select/select.scss

```diff
@@ -53,26 +75,60 @@
         color: helpers.color('content-disabled');
       }
     }
+  }
 
-    &.-borderless {
-      @include mixins.with-inner-focus('action');
-      @include with-padding(); // no border width because border will be unset
+  .ods-select-custom.-open + .ods-icon {
+    transform: rotate(180deg);
```

Comment:
💅🏻  [nitpick]

We should rotate a single axis, small perf boost, either Y or Z I believe.